src/v/redpanda/debug_bundle.cc

```diff
@@ -36,13 +39,46 @@ debug_bundle::rpk_consumer::operator()(tmp_buf buf) {
         std::getline(ss, line);
         if (line.starts_with("Debug bundle saved to")) {
             vlog(bundle_log.trace, "Stop condition reached on line {}", line);
+            status = debug_bundle_status::complete;
             return make_ready_future<consumption_result_type>(
               stop_consuming_type({}));
         }
     }
     return make_ready_future<consumption_result_type>(ss::continue_consuming{});
 }
 
+constexpr std::string_view to_string_view(debug_bundle_status bundle_status) {
+    switch (bundle_status) {
+    case debug_bundle_status::initialized:
+        return "initialized";
+    case debug_bundle_status::running:
+        return "running";
+    case debug_bundle_status::complete:
+        return "complete";
+    }
+    return "unknown";
```

Comment:
I think we already talked about this but probably just want a "running/not-running" state

Reply:
https://github.com/redpanda-data/redpanda/pull/10488#discussion_r1194063813

Reply:
I would agree with you if `HEAD /debug/bundle/status` was **only** supposed to return a running/not-running status. However, that is not the case since that endpoint does not return a binary status.